models/activities/action.go

```diff
@@ -223,6 +223,43 @@ func (a *Action) GetRepoAbsoluteLink() string {
 	return setting.AppURL + url.PathEscape(a.GetRepoUserName()) + "/" + url.PathEscape(a.GetRepoName())
 }
 
+// GetCommentHTMLURL returns link to action comment.
+func (a *Action) GetCommentHTMLURL() string {
```

Comment:
Can we get rid of the term "HTMLURL"? It's just confusing. It's just a absolute URL, right? Why not call it that?

Reply:
To align places, I think we can rename them all in another PR

Reply:
Agree to keep it at the moment. There are really a lot of "HTMLURL" in old code. It's not worth  to make them inconsistent by this PR, it's better to change them together in the future.